src/components/v5/shared/RichText/RichText.tsx

```diff
@@ -52,7 +46,23 @@ const RichText: FC<RichTextProps> = ({
           </div>
         </>
       ) : (
-        notFormattedContent
+        <>
+          <span
+            className={clsx(
+              characterCount >= MIN_ANNOTATION_NUM,
+              'line-clamp-2 text-left',
+            )}
+          >
+            {notFormattedContent}
+          </span>
+
+          {characterCount > NUMBER_OF_CHARS_IN_TWO_LINES &&
```

Comment:
Is there no option to collapse it?

Reply:
there is, you can do it by clicking toggleOffDecriptionSelect on "show less" button